examples/oauth2/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '2'
+services:
+  gateway:
+    image: apicast-test:latest
```

Comment:
Yes, makes sense. I guess it was like that before because there were some fixes for the apicast needed for this to work properly. Not relevant now, as the fixes are already in master.